edx_dl/edx_dl.py

```diff
@@ -173,7 +174,7 @@ def _get_initial_token(url):
     return ''
 
 
-def get_available_sections(url, headers):
+def get_available_sections(course, headers):
```

Comment:
This refactoring is temporal (it goes again my consistency comment of some PRs ago, but I had my reasons to put it there, however this will go back to 'normal' once I finish my next PR. Thanks for accepting this part like this for the moment.